velox/sdk/cpp/jni/JniUtil.h

```diff
@@ -0,0 +1,61 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
please use the correct license header. In general, please also review our coding style document and ensure your code closely follow it:

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

Reply:
done